internal/pkg/print/settings.go

```diff
@@ -6,6 +6,10 @@ type Settings struct {
 	// scope: Markdown
 	EscapeCharacters bool
 
+	// InjectOutputValues inject output values generated by `terraform output` (default: "")
+	// scope: Global
+	InjectOutputValues string
```

Comment:
Everything else is a bool. I would make this a bool and only use the actual filename in the CreateModule function

Reply:
Could you elaborate on:

> only use the actual filename in the CreateModule function

If we change this to a bool then people:
- won't be able to specify which file contains their outputs
- will be forced to use whatever filename we dictate

Reply:
Yes, but in the `cmd`, you can pass the actual filename to CreateModule. It is the only place that it is being used